mclazy.py

```diff
@@ -98,10 +98,14 @@ def sync_to_rawhide_branch(module, pkg_cache, args):
         return
 
     # Build the package
-    rc = run_command (pkg_cache, ['fedpkg', 'build', '--nowait'])
-    if rc != 0:
-        log_error(module, "build")
-        return
+    if not args.no_build:
```

Comment:
This is a good fix. It's also not terribly related to the rest of your commit here. Worth mentioning in the commit message, at least.